actions/Send_Gmail_MOD

```diff
@@ -0,0 +1,202 @@
+module.exports = {
```

Comment:
This file doesn't have an extension (.js)

Reply:
Removing send gmail mod 👍